proposals/notary + tuf.adoc

```diff
@@ -0,0 +1,84 @@
+== Notary & TUF Proposal
+
+*Name of project:* Notary & TUF
+
+*Description:*
+
+The Update Framework (TUF) is a specification designed to solve specifically provenance and trust problems as part of a larger distribution framework.
+
+Notary is a content signing framework implementing the TUF specification in the Go language. The project provides both a client, and a pair of server applications to host signed metadata and perform limited online signing functions. It is the de facto image signing framework in use by Docker, Quay, VMWare, and others.
+
+Notary and TUF have been presented at [CNCF TOC meeting 6/20/2017](https://docs.google.com/presentation/d/1MvCZytMQpTgGW4IvJ1cM0hvnIr8IowH7hFaeXJZ6cp4/edit#slide=id.g2309ce468a_22_0)
+
+*Sponsor / Advisor from TOC:* Solomon Hykes
+
+*Preferred maturity level:* incubating
+
+*Unique identifier:*
+
+* Notary: notary
+* The Update Framework: tuf
+
+*License:*
+
+* Notary: Apache 2.0 
+* TUF: MIT
+
+*Source control repositories:*
+
+* https://github.com/docker/notary
+* https://github.com/theupdateframework/tuf
+* https://github.com/theupdateframework/taps 
+
+*Initial Committers:*
+
+* https://github.com/docker/notary/blob/master/MAINTAINERS
+* https://github.com/theupdateframework/tuf/blob/develop/AUTHORS.txt 
+
+*Infrastructure requirements (CI / CNCF Cluster):*
+
+* CircleCI
+* CodeCov
+* Travis CI
+
+*Issue tracker:*
+
+* https://github.com/docker/notary/issues
+* https://github.com/theupdateframework/tuf/issues
+
+*Mailing lists:*
+
+* Slack: https://dockercommunity.slack.com/messages/notary
+* Google Groups: https://groups.google.com/forum/#!forum/theupdateframework 
+
+*Website:*
+
+* TUF: https://theupdateframework.github.io/
+
+*Release methodology and mechanics:*
+
+* Feature based releases
+
+*Social media accounts:* None
+
+*Existing sponsorship:* Docker for Notary, National Science Foundation and NYU for TUF
+
+*Contributor statistics:*
+
+The notary community is growing slowly with a very small but active base and a larger group of occasional contributors. Maintainers are from Docker, CoreOS and Huawei.
+
+TUF maintainers from NYU and CoreOS.
+
+*Adopters:* Docker, Quay, Huawei, Motorola Solutions, VMWare
+
+*External Dependencies:*
+
+* https://github.com/docker/notary/blob/master/vendor.conf 
+* Protobuf
+* GRPC
+* MySQL/PostgreSQL/rethinkDB
+* https://github.com/yubico/yubico-piv-tool 
+
+*Statement on alignment with CNCF mission:*
+
+Notary is the most secure and widely adopted implementation of The Update Framework to date, and represents a critical security building block for ensuring the provenance and integrity of data in the field of cloud-native computing. As an implementer of The Update Framework it can provide its guarantees over any arbitrary digital content, making it ultimately flexible to any use case requiring security guarantees against attacks up to and including nation state level.
```

Comment:
> I don't have a comparative analysis for you

So then do you agree that the statement about it being "the most secure TUF implementation" is not substantiated? Do you mind removing it or reworking this paragraph then?

> As far as how generic it is, notary itself has been maintained as a vanilla TUF implementation. You'll find nothing in its code that ties it to container images or any other specific type of target.

I believe you when you tell me it's generic enough to not be tied to container images, but I'm asking for an example of it being used in that way. The reason I'm asking is because it's current main use (and all of its documentation that I could skim through describes) container image signing. This is a topic that is next on the list of things to be discussed within the OCI, and if Notary's only real use at the moment is image signing it feels too early to include it (given that there are many other ways that image signing is done by AppC/Flatpak/etc).

>  As noted above, Notary and the Python TUF implementations interoperate.

I am still struggling to understand why both projects are being proposed at the same time. TUF has a reference implementation (which is being pushed as part of this proposal), and an alternate implementation (Notary) is being pushed as well. They both interoperate, so only one implementation is required to live in CNCF and surely any other implementations can be maintained separately.

More importantly, I think that discussions about including TUF itself should be taken separately to discussions about including Notary. For example: At the moment, I am not convinced about the inclusion of Notary (for the reasons discussed) but I am fairly happy with including TUF. If we proceed to a vote, I would be forced to vote against this proposal because I am not yet convinced about including Notary (despite the fact I would want TUF to be included). I don't think I have voting rights, but I would be surprised if no TOB member felt this way (or even the inverse).